cli.js

```diff
@@ -4,18 +4,34 @@ const meow = require('meow');
 const hasYarn = require('has-yarn');
 const logSymbols = require('log-symbols');
 
-meow(`
+const {flags} = meow(`
 	Usage
 	  $ has-yarn
 	  ✔ This project uses Yarn
 
+	Options
+	  --quiet, -q
```

Comment:
Needs a description.

You also need to update the readme output.

Reply:
Sure, I can add one. I just figured it was self-documenting ;)

Will update README as well.